test/fit_tests/common/test_api_utils.py

```diff
@@ -0,0 +1,521 @@
+# Copyright 2016, EMC, Inc.
+
+"""
+ RackHD 1.1 common test utlitiies
+
+ authors: ehohenstein, yuchi zhang
+
+ Purpose/intent
+ This utility library contain helper functions for parsing response data from the RackHD APIs.
+"""
+
+import sys
+import subprocess
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/common")
+import fit_common
+
+import unicodedata
+import json
+
+##########################################
+#  Node List Utils
+##########################################
+
+def rackhd_get_node_list():
```

Comment:
@hohene the on-http git repository includes bits to generate two additional debian packageswith a 1.1 and 2.0 python library from swagger definition files. Packages are pushed to bintray as well. Probably not the ideal place to store libraries for RackHD, but since they update with this repository, we've been using it to date for the current tests in https://github.com/rackhd/rackhd tests